tests/DevApps/daemon-app/daemon-console-calling-downstreamApi/DaemonConsoleCallingDownstreamApi.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>net7.0</TargetFrameworks>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\minimal-web-api\WeatherForecast.cs" Link="WeatherForecast.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(UseWip)' == ''">
```

Comment:
We can either build from NuGet / artifact feed, or from the solution.

Would it make sense to have an environment variable in the Directory.Build.Props with the version of Id.Web to test?

Reply:
Yes, but you dont need to add it to this PR.